gap/oper.gi

```diff
@@ -1684,3 +1684,51 @@ function(digraph, list)
 
   return DigraphShortestDistance(digraph, list[1], list[2]);
 end);
+
+# Partial Order Digraphs
+
+InstallMethod(PartialOrderDigraphJoinOfVertices,
+"for a digraph and two positive integers",
+[IsDigraph, IsPosInt, IsPosInt],
+function(digraph, i, j)
+  local x, nbs, intr;
+
+  if not IsPartialOrderDigraph(digraph) then
+    ErrorNoReturn("Digraphs: PartialOrderDigraphJoinOfVertices: usage,\n",
+                  "the first argument <digraph> must be a partial order",
+                  "digraph,");
+  fi;
+
+  nbs := OutNeighboursMutableCopy(digraph);
+  intr := Intersection(nbs[i], nbs[j]);
+  for x in intr do
+    if intr = Set(nbs[x]) then
+      return x;
+    fi;
+  od;
+
+  return fail;
+end);
+
+InstallMethod(PartialOrderDigraphMeetOfVertices,
+"for a digraph and two positive integers",
+[IsDigraph, IsPosInt, IsPosInt],
+function(digraph, i, j)
+  local x, nbs, intr;
+
+  if not IsPartialOrderDigraph(digraph) then
+    ErrorNoReturn("Digraphs: PartialOrderDigraphMeetOfVertices: usage,\n",
+                  "the first argument <digraph> must be a partial order",
```

Comment:
there should be a space after "order" here, since the word "digraph" on the line below doesn't have a space before it. As things stand, this will print "...partial orderdigraph,"